panels/config/config-entries/ha-form.html

```diff
@@ -17,6 +17,9 @@
     <template is='dom-if' if='[[_isArray(schema)]]' restamp>
       <template is='dom-repeat' items='[[schema]]'>
         <ha-form
+          hass='[[hass]]'
+          domain='[[domain]]'
```

Comment:
I would love to keep this component more generic. It should not be bound to just config flows. What about we allow passing in the computeLabel and computeError functions as properties?